### PR TITLE
Fasih/ removed legacy way to fetch data and used useWS to fetch data

### PR DESCRIPTION
**Pull request by fasihali-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/store/website-status-context.tsx [hunk 1]
```diff
@@ -26,30 +26,26 @@ export const WebsiteStatusProvider = ({ children }: WebsiteStatusProviderProps)
     const WEBSITE_STATUS_COUNTRY_KEY = 'website_status'
     const COOKIE_EXPIRY_DAYS = 7
 
-    const [website_status, setWebsiteStatus] = useCookieState(WEBSITE_STATUS_COUNTRY_KEY, {
+    const [, setWebsiteStatus] = useCookieState(WEBSITE_STATUS_COUNTRY_KEY, {
```
Comment (mitra-deriv) [inline src/store/website-status-context.tsx:29]:
```suggestion
    const [setWebsiteStatus] = useCookieState(WEBSITE_STATUS_COUNTRY_KEY, {
```

Comment (mitra-deriv) [inline src/store/website-status-context.tsx:29]: @fasihali-deriv Why do we need the comma here? 👀

**Review by mitra-deriv — APPROVED** [review]

**Review by habib-deriv — APPROVED** [review]
LGTM

File: src/store/website-status-context.tsx (hunk 1 above)
Comment (fasihali-deriv) [reply src/store/website-status-context.tsx:29]: Actually I am destructuring an array here and I only need second index that's why I skipped first index by just adding comma.

**Review by habib-deriv — APPROVED** [review]
LGTM

**Pull request merged**